MAUI/DataGrid/column-resizing.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Column Resizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column reizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Resizing columns in MAUI DataGrid (SfDataGrid)
+
+The `SfDataGrid` allows you to resize the columns by tapping and dragging the right border of the column headers. Resizing can be enabled or disabled by setting the `SfDataGrid.AllowResizingColumns` property. A resizing indicator is displayed while resizing a column.
+
+N> Resizing considers `DataGridColumn.MinimumWidth` and `DataGridColumn.MaximumWidth` of the column and will not exceed the minimum and maximum width constraints.
+
+{% tabs %}
+{% highlight xaml %}
+
+    <syncfusion:SfDataGrid  x:Name="dataGrid"
```

Comment:
Start codes without providing front spacing. Follow this in all the code snippets and each lines,

<img width="313" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/60499432/67c7b959-2191-4ed9-8f42-4a74c65f76ac">

Reply:
Addressed the review concerns provided.